.github/workflows/style.yaml

```
@@ -75,6 +75,9 @@ jobs:
     - name: Check Policies
       run: scripts/ci/jobs/policy-checks.sh
 
+    - name: Check Konflux pipeline modifications
```

Let's simplify it, please:
```suggestion
    - name: Check Konflux pipelines
```